oauth2client/service_account.py

```diff
@@ -461,3 +462,56 @@ def create_delegated(self, sub):
         result._private_key_pkcs12 = self._private_key_pkcs12
         result._private_key_password = self._private_key_password
         return result
+
+
+class SelfSignedJWTCredentials(object):
```

Comment:
JWTs are always self-signed whether you use them directly (like here) or exchange them for an oauth2 access token. Here are some suggestions to rename them:
- JWTAccessCredentials (good)
- JWTScopelessCredentials (not as good).